packages/google_sign_in/google_sign_in_web/README.md

```diff
@@ -0,0 +1,66 @@
+# google_sign_in_web
```

Comment:
should we give instructions on how to add a dependency on this plugin? I don't think we're able to have `google_sign_in` automatically pull in `google_sign_in_web`, yet

Reply:
Yes, I added this under the "Import the package" section of the README.md. I also moved the Import the package section above the Web integration one. Thanks!